CHANGELOG.md

```diff
@@ -16,9 +16,13 @@ All user visible changes to this project will be documented in this file. This p
 - UI:
     - Home page:
         - Own avatar displaying with question marks when signing in or signing up. ([#978], [#950])
+    - Chat page:
+        - Tapping on image opens gallery when being in context menu. ([#983], [#979])
```

Comment:
>  [ ] [CHANGELOG entry][l:3] is added (if required)

Why don't you do it? It's your responsibility to add changelog entries for end-user visible changes.